README.md

```diff
@@ -1,3 +1,5 @@
+# [tedee Release Notes](https://tedee-com.github.io/tedee-release-notes/)
```

Comment:
Add an introduction section where you describe what is this repo about. It contains release history for Tedee Android and iOS app, firmware changes for Tedee devices. The release notes are available here.

Reply:
done